code/__DEFINES/jobs.dm

```diff
@@ -418,3 +418,84 @@
 		if(jobname in hud_to_department_acc)
 			return hud_to_department_acc[jobname]
 	return ACCOUNT_CIV
+
+// used to determine chat color by HUD in `chatmessage.dm`
+// Note: custom colors are what I really didn't put much attention into. feel free to change its color when you feel off.
+/datum/chatmessage/proc/get_chatcolor_by_hud(jobname)
+	if(jobname)
+		var/static/hud_to_chatcolor = list(
+			// Command
+			JOB_HUD_RAWCOMMAND = "#ffecca", // custom command color
```

Comment:
A few things.
First, it's better for them to be defined (color strings).
Also the styling is inconsistent, some are lower, some are upper case (make them all upper).

Reply:
changed. but note that these upper cases will become the lower case when you just change it from picking up a color window from visual studio.